apps/nrm/views/check-your-answers-so-far.html

```diff
@@ -0,0 +1,11 @@
+{{<partials-page}}
+  {{$page-content}}
+    {{#hasReports}}
+      {{> partials-check-your-answers-so-far-summary-table}}
+      {{#input-submit}}continue{{/input-submit}}
+    {{/hasReports}}
+    {{^hasReports}}
+    Error
```

Comment:
what scenario would this appear?

Reply:
If for whatever reason the session gets destroyed or the user goes directly to the url without going through the app?

Is this a bit too much?

Reply:
I think it's fine, we will just test it